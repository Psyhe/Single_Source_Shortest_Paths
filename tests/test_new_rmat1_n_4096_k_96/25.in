4096 1075 1117
1075 320 118
1075 340 233
1075 518 241
1075 1024 170
1075 2120 242
1075 1059 73
1075 192 184
1075 1166 8
1076 516 164
1076 1058 21
1076 689 24
1076 2568 148
1076 1810 132
1076 336 209
1076 2432 29
1076 0 220
1076 35 194
1076 3 201
1076 225 54
1076 722 157
1076 66 178
1076 736 138
1076 552 55
1076 137 232
1076 98 12
1076 2565 68
1076 0 116
1076 144 103
1077 2560 252
1077 2621 159
1077 1024 70
1077 1672 35
1077 2216 240
1077 2068 229
1077 256 165
1077 321 32
1077 0 137
1078 640 83
1078 769 40
1078 534 218
1078 1544 102
1078 138 93
1078 2312 43
1078 120 181
1078 25 6
1078 416 179
1079 2368 142
1079 2434 58
1079 12 65
1080 262 114
1080 3136 26
1080 898 75
1080 1610 175
1080 2052 30
1080 68 102
1080 2096 22
1080 2080 30
1080 2121 27
1080 960 219
1080 8 114
1080 624 173
1080 592 19
1080 16 49
1080 2688 156
1080 2626 167
1080 576 190
1081 2112 252
1081 2 77
1081 386 214
1081 1696 109
1081 291 35
1081 288 12
1081 2050 157
1082 100 22
1082 64 16
1082 833 203
1082 1024 227
1082 36 99
1082 400 205
1082 264 26
1082 1664 45
1083 5 110
1083 259 146
1083 416 238
1084 512 71
1084 425 217
1084 2192 116
1084 257 89
1084 576 237
1084 657 76
1084 2058 109
1084 64 79
1085 1024 254
1086 4 193
1086 10 25
1086 172 159
1087 2568 186
1088 2116 93
1088 197 180
1088 134 16
1088 2050 115
1088 560 179
1088 354 87
1088 3332 212
1088 138 165
1088 621 244
1088 8 231
1088 2102 41
1088 8 125
1088 2098 142
1088 3272 249
1088 1072 151
1088 1409 4
1088 657 76
1088 304 175
1088 192 68
1088 70 186
1088 416 18
1088 300 140
1088 640 238
1088 2185 178
1088 224 15
1088 265 46
1088 1035 206
1088 96 74
1088 32 252
1088 2075 97
1088 2058 11
1088 2626 147
1088 128 234
1088 65 70
1088 1280 151
1088 544 81
1088 17 163
1088 418 186
1088 298 31
1088 24 49
1088 1495 203
1088 192 227
1088 272 74
1088 2177 94
1088 2816 15
1088 108 182
1088 2048 62
1088 176 153
1088 529 135
1088 520 236
1088 2373 10
1088 3376 235
1088 2068 126
1088 1217 111
1088 3083 251
1088 320 205
1088 530 230
1088 64 30
1088 1055 82
1088 2304 15
1088 257 212
1088 256 204
1088 2560 255
1088 589 97
1088 290 87
1088 1 37
1088 1104 92
1088 2 51
1088 136 161
1088 160 80
1088 588 126
1088 137 16
1088 2184 171
1088 128 255
1088 1160 175
1088 133 38
1088 129 133
1088 2065 28
1088 288 33
1088 70 65
1088 1346 77
1088 2308 57
1088 2403 188
1088 328 30
1088 1066 117
1088 1098 190
1088 1034 47
1088 1821 26
1088 2058 183
1088 523 44
1088 2328 154
1088 128 109
1088 16 136
1088 656 12
1088 17 38
1088 912 61
1088 3740 106
1088 521 165
1088 768 176
1088 69 151
1088 544 17
1088 305 170
1088 720 38
1088 512 87
1088 901 205
1088 80 243
1088 266 246
1088 18 254
1088 2926 146
1088 624 121
1088 3083 240
1088 3210 149
1088 657 232
1088 2116 213
1088 2456 254
1088 2706 189
1088 64 206
1088 24 26
1088 1089 163
1088 3 198
1088 2130 62
1088 17 67
1088 1712 197
1088 8 115
1088 2833 159
1088 2066 134
1088 2307 196
1088 292 199
1088 1285 110
1088 64 137
1088 2608 236
1088 2448 139
1088 2688 59
1088 2080 40
1088 1284 82
1088 12 248
1088 48 39
1088 2133 161
1088 1417 50
1088 2195 187
1088 3328 159
1088 1153 91
1088 2176 2
1088 72 96
1088 2048 88
1088 259 190
1088 640 16
1088 2120 74
1088 2113 5
1088 2050 138
1088 72 47
1088 2 223
1088 0 14
1088 136 36
1088 78 31
1088 16 255
1088 1024 43
1088 1030 238
1088 6 25
1088 68 18
1088 170 165
1088 2232 35
1088 32 124
1088 7 67
1088 2314 9
1088 266 117
1088 1030 112
1088 2700 155
1088 2064 48
1088 514 32
1088 1 63
1088 83 66
1088 2064 52
1088 384 155
1088 88 170
1088 96 11
1088 547 17
1088 1024 226
1088 1287 47
1088 392 134
1088 1641 120
1088 2368 22
1088 335 141
1088 1216 26
1088 8 128
1088 1560 166
1088 512 11
1088 37 147
1088 512 68
1088 1431 111
1088 1536 15
1088 2176 84
1088 1024 7
1088 272 105
1088 162 205
1088 3384 19
1088 1 153
1088 2576 73
1088 0 145
1088 2233 167
1088 544 96
1088 2 61
1088 2178 84
1088 3376 26
1088 64 57
1088 3072 46
1088 280 60
1088 3586 27
1088 2048 101
1088 2764 218
1088 464 76
1088 1028 10
1088 520 11
1088 2220 188
1088 806 31
1088 3869 53
1088 1316 217
1088 304 150
1088 1972 171
1088 138 75
1088 2147 68
1088 520 129
1088 1281 176
1088 2048 199
1088 832 163
1088 152 30
1088 2148 139
1088 3907 8
1088 301 100
1088 5 91
1088 2049 241
1088 3921 121
1088 66 156
1088 2566 199
1088 2176 97
1088 1206 10
1088 2088 224
1088 21 173
1088 1 56
1088 2592 54
1088 291 254
1088 2560 225
1088 256 207
1088 348 93
1088 1040 49
1088 1666 56
1088 3205 47
1088 2097 79
1088 898 243
1088 39 68
1088 2068 68
1088 2848 87
1088 2728 143
1088 96 252
1088 3209 123
1088 2624 64
1088 1032 196
1088 1130 60
1088 1312 184
1088 3664 142
1088 17 179
1088 8 251
1088 256 8
1089 16 72
1089 17 253
1089 2312 116
1089 808 99
1089 1 220
1089 2 122
1089 0 53
1089 0 216
1089 48 17
1089 2073 149
1089 3 201
1089 1024 13
1089 2500 27
1089 264 114
1089 260 79
1089 1411 65
1089 630 254
1089 16 20
1089 512 123
1089 1113 130
1089 2500 239
1089 3085 245
1089 274 46
1089 2818 121
1089 12 109
1089 3746 15
1089 2368 133
1089 2 70
1089 2144 118
1089 160 129
1089 324 158
1089 18 250
1089 328 242
1089 16 102
1089 256 185
1089 40 23
1089 2060 232
1089 2122 254
1089 268 12
1089 8 115
1089 545 56
1089 76 172
1089 2694 224
1089 38 75
1089 896 220
1089 1186 119
1089 512 31
1089 288 246
1089 1794 167
1089 146 151
1089 801 16
1089 1294 112
1089 3076 79
1089 618 22
1089 8 67
1089 3593 177
1089 2066 172
1089 544 88
1089 642 222
1089 1242 215
1089 1088 163
1089 260 171
1089 70 47
1089 2113 189
1089 151 53
1089 320 212
1089 0 215
1089 546 159
1089 2192 133
1089 1040 5
1089 2838 127
1089 3481 114
1089 288 105
1089 401 92
1089 3089 76
1090 163 141
1090 2440 219
1090 2432 35
1090 3075 162
1090 2324 191
1090 138 15
1090 152 80
1090 132 235
1090 4 111
1090 1108 68
1090 9 138
1090 2208 45
1090 2240 236
1090 32 220
1090 324 141
1090 8 98
1090 2060 166
1090 9 57
1090 22 81
1090 3284 126
1090 1064 169
1090 557 208
1090 1 253
1090 3072 248
1090 257 229
1090 1283 88
1090 2112 60
1090 552 161
1090 2565 169
1090 4 149
1090 392 101
1090 2368 40
1090 1201 250
1090 0 14
1090 4 247
1090 768 55
1090 34 178
1090 65 56
1090 1038 14
1090 39 99
1090 288 132
1090 2185 152
1090 1712 66
1090 1171 202
1090 1024 157
1090 385 53
1090 130 22
1090 0 222
1090 675 97
1090 2336 25
1090 3114 26
1090 1177 72
1090 3400 99
1090 2156 142
1090 73 178
1090 1614 248
1090 1104 253
1090 273 199
1090 1184 84
1090 0 235
1090 2230 120
1090 100 96
1090 2048 134
1090 0 109
1090 64 175
1090 1024 207
1090 1044 225
1091 280 161
1091 257 113
1091 788 74
1091 2136 96
1091 136 196
1091 467 49
1091 518 124
1091 38 97
1091 0 73
1091 3594 133
1091 532 79
1091 44 201
1091 0 45
1091 32 130
1091 640 23
1091 1412 213
1091 1044 94
1091 2177 180
1091 2572 242
1091 272 47
1091 200 45
1091 131 107
1091 1 124
1091 2697 201
1091 304 19
1091 48 64
1091 1280 182
1092 65 154
1092 2052 120
1092 2246 230
1092 2898 23
1092 532 200
1092 92 68
1092 2192 43
1092 2052 92
1092 2052 226
1092 452 46
1092 2072 217
1092 6 132
1092 17 187
1092 768 12
1092 36 173
1092 8 166
1092 276 172
1092 1176 112
1092 281 69
1092 266 5
1092 3136 93
1092 1026 75
1092 40 241
1092 1152 165
1092 313 26
1092 2176 88
1092 517 129
1092 4 107
1092 544 53
1092 768 216
1092 2721 197
1092 1314 16
1092 16 115
1092 192 158
1092 1201 237
1092 960 122
1092 233 143
1092 297 46
1092 3080 247
1092 2117 206
1092 772 199
1092 64 226
1092 12 93
1092 1047 14
1092 577 250
1092 3074 1
1092 4 234
1092 201 164
1092 2569 202
1092 7 176
1092 64 15
1092 519 91
1092 481 120
1092 1318 217
1092 784 117
1092 0 169
1092 129 122
1092 2450 206
1092 528 239
1092 16 121
1092 137 237
1092 3077 68
1092 192 241
1092 1106 156
1092 520 6
1092 2131 254
1092 352 109
1092 2480 52
1092 2816 245
1093 1344 179
1093 1032 232
1093 1024 231
1093 2126 179
1093 2224 20
1093 2392 235
1093 2368 7
1093 1540 247
1093 2576 184
1093 490 111
1093 16 164
1093 1040 140
1093 76 224
1093 1824 48
1093 64 117
1093 2053 60
1093 8 250
1093 26 61
1093 658 100
1093 400 91
1093 0 137
1094 2712 0
1094 64 196
1094 8 72
1094 2560 65
1094 2100 137
1094 2 93
1094 10 70
1094 512 232
1094 1664 149
1094 224 68
1094 3883 9
1094 4 156
1094 100 81
1094 2048 111
1094 3076 126
1094 64 2
1094 2624 117
1094 704 115
1094 269 163
1095 2576 192
1095 513 191
1095 0 182
1095 872 209
1095 1026 50
1095 914 175
1095 529 70
1095 2124 87
1095 544 156
1095 256 135
1095 2080 174
1096 1665 136
1096 17 210
1096 1242 255
1096 172 104
1096 1040 72
1096 3393 18
1096 1056 0
1096 522 48
1096 514 187
1096 690 149
1096 16 18
1096 584 235
1096 642 169
1096 1026 53
1096 592 29
1096 2089 148
1096 2085 56
1096 2056 204
1096 273 20
1096 4 138
1096 652 44
1096 2048 158
1096 2119 159
1096 2184 76
1096 1280 211
1096 2 159
1096 897 237
1096 2052 108
1096 1024 16
1096 2 200
1096 1472 129
1096 2048 73
1096 516 47
1096 18 130
1096 0 16
1096 649 11
1096 769 36
1096 5 234
1096 598 57
1096 1153 36
1096 2067 55
1096 2760 135
1096 2048 90
1096 3072 149
1096 592 197
1096 64 152
1096 2064 217
1096 929 203
1096 0 37
1096 140 78
1096 1540 97
1096 68 206
1096 2248 34
1096 1030 119
1096 389 233
1096 528 75
1096 96 95
1096 536 107
1096 0 135
1096 2563 155
1096 513 54
1096 20 68
1096 1256 58
1096 1028 53
1096 42 136
1096 33 80
1096 6 85
1096 3106 13
1096 832 137
1096 1610 0
1096 592 223
1096 1152 12
1096 1 144
1096 3616 61
1096 72 96
1097 86 120
1097 2689 242
1097 1414 86
1097 1573 39
1097 659 128
1097 320 211
1097 64 227
1097 3074 189
1097 514 122
1097 2881 22
1097 177 17
1097 328 32
1097 295 92
1097 1152 83
1097 552 179
1097 132 202
1097 130 247
1097 2053 228
1097 134 196
1097 544 236
1097 3 2
1097 18 127
1097 528 124
1097 512 30
1097 64 165
1097 1413 112
1097 640 89
1097 128 85
1098 259 40
1098 708 141
1098 136 151
1098 740 220
1098 2112 139
1098 0 182
1098 593 188
1098 512 20
1098 696 30
1098 740 115
1098 2058 99
1098 277 4
1098 1552 2
1098 1281 58
1098 12 231
1098 1025 180
1098 1152 227
1098 7 184
1098 1088 190
1098 1552 182
1098 2883 202
1099 56 145
1099 210 51
1099 6 90
1099 642 69
1099 42 31
1100 896 64
1100 512 131
1100 3092 115
1100 2136 208
1100 3136 1
1100 3009 39
1100 272 39
1100 656 163
1100 1050 166
1100 32 18
1100 2072 5
1100 144 58
1100 0 239
1100 2608 67
1100 169 120
1100 3632 104
1100 2345 43
1100 2946 241
1100 128 201
1100 560 236
1100 770 156
1100 42 116
1100 1114 190
1100 15 235
1100 1128 39
1100 2705 58
1100 2608 137
1101 3104 76
1101 87 34
1101 256 209
1101 2240 50
1101 676 64
1101 656 144
1101 1 36
1101 1168 227
1102 226 152
1102 256 79
1102 384 236
1102 9 182
1102 263 44
1102 47 66
1103 2432 217
1104 2572 114
1104 2598 193
1104 1066 145
1104 2048 35
1104 3445 214
1104 2 30
1104 3344 219
1104 324 4
1104 29 247
1104 1153 159
1104 1289 75
1104 384 149
1104 2452 214
1104 512 208
1104 1176 52
1104 9 22
1104 1 25
1104 171 66
1104 262 28
1104 664 36
1104 3080 7
1104 512 237
1104 2 165
1104 16 254
1104 1056 178
1104 838 157
1104 1088 92
1104 33 88
1104 1255 22
1104 2176 140
1104 1164 71
1104 102 128
1104 3136 241
1104 136 7
1104 2305 130
1104 192 1
1104 64 31
1104 897 91
1104 3072 187
1104 526 46
1104 32 6
1104 321 190
1104 3216 101
1104 2072 25
1104 2281 122
1104 1090 253
1104 172 173
1104 1284 160
1104 1024 133
1104 22 164
1104 48 125
1104 768 68
1104 35 58
1104 2052 209
1104 80 159
1104 520 31
1104 74 135
1104 64 126
1104 512 120
1104 257 129
1104 160 119
1104 769 46
1104 519 93
1104 2134 79
1104 50 156
1104 1616 254
1104 1049 79
1104 256 20
1105 656 255
1105 3097 173
1105 2057 152
1105 1322 96
1105 512 102
1105 2069 171
1105 2112 77
1105 2625 134
1105 3154 31
1105 1408 158
1105 553 145
1105 3168 245
1105 2304 106
1105 64 36
1105 22 221
1105 276 118
1105 517 5
1105 1162 134
1105 4 107
1105 1248 97
1105 512 113
1105 1028 234
1105 1414 91
1105 640 10
1105 308 214
1106 268 149
1106 2440 0
1106 1024 5
1106 3140 5
1106 336 92
1106 9 79
1106 1057 47
1106 602 219
1106 2562 31
1106 1568 206
1106 520 123
1106 2 251
1106 1092 156
1106 2097 248
1106 768 31
1106 1040 48
1106 0 165
1106 357 164
1106 310 145
1106 12 54
1106 1416 222
1106 2069 100
1106 5 83
1106 2587 76
1106 160 156
1107 320 70
1107 2208 156
1107 3594 221
1107 2176 42
1107 44 143
1107 160 209
1108 1 129
1108 32 68
1108 2 155
1108 1 20
1108 1090 68
1108 1060 125
1108 546 229
1108 80 132
1108 82 22
1108 194 86
1108 66 11
1108 1576 201
1108 0 135
1108 258 153
1108 1412 177
1108 512 34
1108 1050 145
1108 2292 12
1108 3752 141
1108 2048 87
1108 1059 45
1108 776 164
1108 544 144
1108 653 12
1108 3072 127
1108 57 130
1108 512 186
1108 2192 120
1109 353 221
1109 0 162
1109 1200 254
1109 2244 160
1109 3432 180
1109 2049 88
1109 2697 242
1109 32 123
1109 520 43
1109 2113 83
1110 785 66
1110 2085 225
1110 16 159
1110 2093 91
1110 2094 117
1110 610 208
1110 2048 11
1110 596 132
1110 2128 245
1111 2336 79
1111 512 94
1111 2121 158
1111 70 86
1111 128 158
1112 96 122
1112 192 78
1112 744 116
1112 512 43
1112 3585 251
1112 256 24
1112 676 77
1112 3105 125
1112 271 210
1112 904 105
1112 520 205
1112 1600 178
1112 522 54
1112 142 63
1112 3072 203
1112 144 8
1112 2688 140
1112 16 51
1112 1024 136
1112 1544 202
1112 240 82
1112 2432 63
1113 2665 88
1113 198 252
1113 136 62
1113 288 113
1113 564 202
1113 1792 49
1113 8 74
1113 1321 126
1113 386 37
1113 1089 130
1113 36 199
1114 1545 166
1114 2348 88
1114 2081 196
1114 1600 244
1114 512 182
1114 1100 190
1114 128 164
1114 16 175
1114 673 243
1114 2056 205
1114 2224 123
1114 1072 127
1114 432 40
1114 0 180
1114 2088 170
1115 272 90
1115 2 160
1116 3 43
1116 552 84
1116 1024 59
1116 1672 185
1116 135 91
1116 838 249
1116 0 146
1116 2100 79
1117 324 180
1117 512 77
