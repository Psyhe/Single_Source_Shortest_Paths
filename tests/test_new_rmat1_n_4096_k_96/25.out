42
17
36
17
66
22
13
17
112
75
255
31
189
3
8
14
25
6
10
3
55
4
6
7
34
7
37
75
221
5
11
5
43
16
45
12
81
11
50
81
95
47
81
